### PR TITLE
Typo in the parameter `visible` in `templates.py`

### PR DESCRIPTION
**Pull request by abidlabs**

### CODE REVIEW

**Comment by abidlabs** [comment]
Going through the components, I realized the `requires_permissions` parameters is no longer used, so I removed that as well

File: gradio/components.py [hunk 1]
```diff
@@ -104,14 +104,12 @@ def __init__(
         show_label: bool = True,
         interactive: Optional[bool] = None,
         visible: bool = True,
-        requires_permissions: bool = False,
```
Comment (freddyaboulton) [inline gradio/components.py:107]: Why is this deleted?

File: gradio/templates.py [hunk 2]
```diff
@@ -261,7 +261,7 @@ def __init__(
             label=label,
             show_label=show_label,
             interactive=interactive,
-            visibile=visible,
+            visible=visible,
```
Comment (freddyaboulton) [inline gradio/templates.py:264]: 🙈

**Review by freddyaboulton — APPROVED** [review]
Thanks @abidlabs !

File: gradio/components.py (hunk 1 above)
Comment (abidlabs) [reply gradio/components.py:107]: We used to use `requires_permissions` to figure out if a component needed to request permissions in colab and print a message accordingly. We don't do need to do that anymore. (So unrelated to the original reason I made this PR, but when I saw it, I figured I'd just remove it)

**Comment by abidlabs** [comment]
Thanks for reviewing @freddyaboulton!
